src/utils/AutoDiscoveryUtils.js

```diff
@@ -67,7 +67,7 @@ export default class AutoDiscoveryUtils {
                 {}, {
                     a: (sub) => {
                         return <a
-                            href="https://github.com/vector-im/riot-web#configjson"
+                            href="https://github.com/vector-im/riot-web/blob/master/docs/config.md"
```

Comment:
it's a bit unfortunate that for the next week or so this will 404, but that's fine.

Reply:
Ah yeah... I guess only errors on develop will see a broken link until next release... So yeah, probably okay.